src/unity/lib/annotation/build/format/obj_c/Data.pbobjc.m

```diff
@@ -0,0 +1,285 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Why do we have `pbobjc` files generated and committed? Nothing appears to be using them.

Reply:
Removed the files!